earthkit/geo/latlon.py

```diff
@@ -0,0 +1,63 @@
+# (C) Copyright 2024 ECMWF.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+#
+
+from . import constants
+
+
+def generate_latlon(dx, dy):
```

Comment:
I wonder if the name should be a bit more explicit that it will generate complete matrices of lats/lons rather than a 1d array of each. Something like generate_latlon_grids or generate_latlon_matrices (still trying to think of the better name)

Reply:
The primary goal of this method is to return the latitudes/longitudes of the global grid generated with MIR interpolation when only dlon/dlat is specified the target grid.

Reply:
It must give correct results to the target lat-lon grids earthkit-regrid offers. This is the current scope. When the geo library will be available this method will simply delegate the call to it and will offer a more generic solution.